### PR TITLE
Update the request to request net-export v1.10.0+ on R14+

### PR DESCRIPTION
**Pull request by ericgraf**
This PR is to change the request parameter to request the next release increase the net-exporter version.

### CODE REVIEW

File: aws/requests.yaml [hunk 1]
```diff
@@ -25,6 +25,9 @@ releases:
         - name: chart-operator
           version: ">= 2.12.0"
           issue: https://github.com/giantswarm/giantswarm/issues/15213
+        - name: net-exporter
```
Comment (MarcelMue) [inline aws/requests.yaml:28]: I think we should put this into it's own bracket which is higher than the currently latest version.

Comment (ericgraf) [reply aws/requests.yaml]: Changed let me know if this is better.

File: aws/requests.yaml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 releases:
+    - name: ">= 14.1.1"
```
Comment (MarcelMue) [inline aws/requests.yaml:2]: I think putting it in the `>= 15.0.0` bracket makes more sense -in general the `validate` CI job should succeed.

Comment (ericgraf) [reply aws/requests.yaml]: changed :)

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**